core/common/src/ClipStyle.ts

```diff
@@ -97,6 +98,78 @@ export class CutStyle {
   }
 }
 
+/** Wire format describing a [[ClipIntersectionStyle]].
+ * @see [[ClipStyleProps.ClipIntersectionStyle]].
+ * @public
+ * @extensions
+ */
+export interface ClipIntersectionStyleProps {
+  /** Color to apply to intersection of geometry and clip planes, default white */
+  color?: RgbColorProps;
+  /** Number of pixels to be considered intersecting the clip plane, default 1 */
+  width?: number;
+}
+
+/** As part of a [[ClipStyle]], describes how to colorize geometry intersecting the clip planes.
+ * @note Edges are highlighted only if [[ClipStyle.ClipIntersectionStyle]] is `true`.
+ * @public
+ * @extensions
+ */
+export class ClipIntersectionStyle {
+  /** Color to apply to intersection of geometry and clip planes, default white */
+  public readonly color?: RgbColor;
```

Comment:
`color` and `width` should not be permitted to be `undefined`.

Reply:
If they are undefined, the constructor will just return the default values (white, one pixel). I left them able to be undefined to have a quicker/cleaner default constructor, e.g. line 136:

`  public static readonly defaults = new ClipIntersectionStyle();`

If this needs to be changed, it certainly can be.

Reply:
An instance of `ClipIntersectionStyle` has a defined color and width. If the JSON props supplied when instantiating the class do not specify either or both of those properties, it's the class's job to initialize them to defaults. Users of the class should not have to check if those properties are undefined and enforce the defaults themselves (clunky and error-prone).

This same pattern is followed through the various DisplayStyle APIs.

Reply:
This has now been addressed, can it be resolved?